examples/examples-cpu/snowflake/snowflake-dask.ipynb

```diff
@@ -0,0 +1,388 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Snowflake + Dask\n",
+    "\n",
+    "How to load data from a Snowflake table or query into a Dask dataframe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to Snowflake\n",
+    "\n",
+    "See [README](README.md) for more details on how to set up the credentials file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import yaml\n",
+    "import snowflake.connector\n",
+    "\n",
+    "creds = yaml.full_load(open('/home/jovyan/snowflake_creds.yml'))\n",
+    "\n",
+    "# get connection info\n",
+    "conn_info = {\n",
+    "    'warehouse': 'COMPUTE_WH',\n",
+    "    'database': 'NYC_TAXI',\n",
+    "    'schema': 'PUBLIC',\n",
+    "    **creds,\n",
+    "}\n",
+    "conn = snowflake.connector.connect(**conn_info)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup query\n",
+    "\n",
+    "We need to set up a query that will return chunks of the full result based on a column in the data. These become our partitions in a Dask dataframe. We use a [binding for the Snowflake query](https://docs.snowflake.com/en/user-guide/python-connector-example.html#binding-data) so that we can pass different values at execution time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query = \"\"\"\n",
+    "SELECT *\n",
+    "FROM taxi_yellow\n",
+    "WHERE\n",
+    "    date(pickup_datetime) = %s\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Validate the query is good with pandas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cur = conn.cursor().execute(query, '2019-01-01')\n",
+    "df = cur.fetch_pandas_all()\n",
+    "len(df), df.memory_usage().sum() / 1e6  # memory size in MB"
```

Comment:
you'd have to already understand what these functions do to understand the two numbers printed here. Could you make this a little more user-friendly?

```python
print(f"number of rows: {len(def)}")
print(f"size in memory (MB): {df.memory_usage().sum() / 1e6}")
```

Reply:
ah yes, will definitely update. Throughout the rest of the examples there's a print in a one-liner and I'll change it to that for consistency